AutoRest/Generators/NodeJS/NodeJS.Tests/typings/mocha/mocha.d.ts

```diff
@@ -0,0 +1,214 @@
+// Type definitions for mocha 2.2.5
+// Project: http://mochajs.org/
```

Comment:
Do we have to generate this typings whenever we update the mocha version?
At the same time, if we take a dependency on a new package then how do we generate typings for that package?
Is this Automated currently?